lib/ext/heroku/command/addons.rb

```diff
@@ -251,7 +256,7 @@ def attach
       case response.status
       when 201
         display("done")
-        action("Setting #{response.body["name"]} vars and restarting #{app}") do
+        action("Setting #{response.body["name"]} and restarting #{app}") do
```

Comment:
>Also, sometimes the attachment won't cause a restart. 

Getting config vars (or none) from the attachments response will be enough for us to decide whether to indicate a restart is happening, right? We could also get the release before and after, but this seems more succinct if it's always right.